packages/thirdweb/src/adapters/ethers6.ts

```diff
@@ -281,7 +281,7 @@ async function fromEthersSigner(signer: ethers6.Signer): Promise<Account> {
  * @returns A promise that resolves to an ethers.js signer.
  * @internal
  */
-async function toEthersSigner(
+export async function toEthersSigner(
```

Comment:
why?

Reply:
it started picking up ethers5 as the 'default' import ethers so this test started breaking. switched the test to use the internal function so i can eplicitely pass ethers6 like we're doing in the ethers5 test.

not 100% sure why it started breaking but it felt shaky to me that there's an implied default import since we're not defnining it anywhere

Reply:
ok